woof-code/rootfs-petbuilds/yambar/usr/share/yambar/config.yml.in

```diff
@@ -0,0 +1,85 @@
+bar:
+  height: 26
+  location: top
+  spacing: 5
+  layer: top
+
+  font: DejaVuSans:pixelsize=14
+
+  foreground: FOREGROUNDff
+  background: BACKGROUNDff
+
+  border:
+    width: 1
+    color: OUTLINEff
+    top-margin: 0
+
+  left:
+    - label:
+        content:
+          - string: { on-click: defaultrun, text: Applications, left-margin: 2 }
+    - label:
+        content:
+          - string: { on-click: pmount, text: Places, left-margin: 2 }
+    - label:
+        content:
+          - string: { on-click: defaultfilemanager, text: 🗄️}
+    - label:
+        content:
+          - string: { on-click: defaultbrowser, text: 🌎️}
+    - label:
+        content:
+          - string: { on-click: defaultterminal, text: 🖥}
+    - foreign-toplevel:
+        content:
+          map:
+            tag: activated
+            values:
+              true:
+                - string: {text: "{title}"}
+
+  right:
+    - battery:
+        name: BAT0
+        poll-interval: 15
+        content:
+          string: {text: "🔋{capacity}%"}
+    - alsa:
+        card: default
+        mixer: Master
+        content:
+          map:
+            tag: online
+            values:
+              false: {string: {text: 🔇}}
+              true:
+                map:
+                  on-click: defaultaudiomixer
+                  tag: muted
+                  values:
+                    true: {string: {text: 🔇}}
+                    false:
+                      ramp:
+                        tag: volume
+                        items:
+                          - string: {text: "🔈{percent}%"}
+                          - string: {text: "🔉{percent}%"}
+                          - string: {text: "🔊{percent}%"}
+    - network:
+        name: wlp3s0
```

Comment:
need to figure out how to un-hardcode this.

Reply:
I think this "widget" need to be replaced with a script that runs `connmanctl monitor` and writes 🌐 or 🚫 every time internet connection is restored or lost on all interfaces. yambar supports this kind of notifications from a shell script.

Reply:
Done, see c7c45c8

Reply:
![disabled](https://user-images.githubusercontent.com/1471149/175244647-1145d6a0-1bf7-4473-91a4-9415bb9f62b9.png)
![ready](https://user-images.githubusercontent.com/1471149/175244650-2d9d759b-ba30-4912-bf06-69f972d6def3.png)